projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/func/LLVMCallUnboxNode.java

```diff
@@ -193,6 +193,11 @@ public LLVMAddress executePointee(LLVMAddress value) {
         public int executePointee(int value) {
             return value;
         }
+
+        @Specialization
+        public long executePointee(long value) {
+            return value;
+        }
```

Comment:
At this point this looks like an identity function, should we just accept `Object` ?